### ISSUE TITLE
[Need Help] Typescript error on gridApi

### ISSUE THREAD

**Issue by MudulOzan** [ceolter/ag-grid#8934]
**I'm submitting a ...**  (check one with "x")
```
[] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[X] support request => see 'Requesting Community Support'
```

![image](https://github.com/user-attachments/assets/186662d1-9eab-43db-907b-8d112220d0da)
![image](https://github.com/user-attachments/assets/d176c385-fe7e-4398-997e-ca0a42b90bb2)

**Providing a Reproducible Scenario**
- Just write the samecode with params is CellClickedEvent

**Current behavior**
- Gives error

**Expected behavior**
- To not to have error 

**Please tell us about your environment:**
VSCode, Win 11

* **AG Grid version:** 32.2.1 

* **Browser:**
- Not relevant to topic

* **Language:** React.js TS Ver: 5.7.0

-----------------------------------------------------------------------

To sum up, I can see on the docs that I can use showContextMenu on grid api and the code is linked through typescript however ts shows error but the code works. We are using enterprise version but our license is outdated so we are in research that what will we encounter if we try to upgrade and since we currently use 23.2.1 it is a long jump so I would have missed something thats why I opened a support request instead of a bug report since also it doesn't break anything

**Comment by MudulOzan** [comment]
@AG-Zoheil oh my bad, I was curious why it shows error even though I opened the related type file then I tried to restart ts server and it is perfectly fine now. I am closing this issue :) Sorry taking your time for this one
